AUTHORS

```diff
@@ -1,5 +1,3 @@
-Martin Aigner <martin@maigner.net>
-Christian Barthel <cbarthel@cs.uni-salzburg.at>
-Christoph Kirsch <ck@cs.uni-salzburg.at>
-Michael Lippautz <michael.lippautz@gmail.com>
-Simone Oblasser <simone.oblasser@cs.uni-salzburg.at>
\ No newline at end of file
+Team: Kernel Panic
```

Comment:
See home work flow description: _Add_ your names. Don't remove previous authors! That's called plagiarism